projects/aca-folder-rules/src/lib/rule-details/actions/rule-action.ui-component.ts

```diff
@@ -189,4 +223,24 @@ export class RuleActionUiComponent implements ControlValueAccessor, OnInit, OnDe
       this.form.enable();
     }
   }
+
+  private parseAspectsToSelectOptions(aspects: Aspect[]): CardViewSelectItemOption<unknown>[] {
+    return aspects
+      .sort(function (a, b) {
+        if (a.label === '' || a.label === null) {
+          return 1;
+        }
+        if (b.label === '' || b.label === null) {
+          return -1;
+        }
+        if (a.label === b.label) {
+          return 0;
+        }
+        return a.label < b.label ? -1 : 1;
+      })
```

Comment:
You can use `localeCompare to simplify this:
```suggestion
      .sort((a, b) => a.label?.localeCompare(b.label) ?? -1)
```

Reply:
Applied, thanks.